README.md

```diff
@@ -61,6 +61,12 @@ elements, etc.). Here's an example:
 </script>
 ```
 
+## `webcomponents-es5-loader.js`
+
+Due to the requirement that Custom Elements be ES6 classes (https://html.spec.whatwg.org/multipage/scripting.html#custom-element-conformance), it may make sense to precompile your elements down to ES5 for wider consumption.
+However, this will conflict with the presence of native Custom Elements in browsers that support them.
```

Comment:
Custom Elements must be ES6 classes (https://html.spec.whatwg.org/multipage/scripting.html#custom-element-conformance). Since most projects need to support a wide range of browsers that don't necessary support ES6, it may make sense to compile your project to ES5. However, ES5-style custom element classes will not work with native Custom Elements because ES5-style classes cannot properly extend ES6 classes, like HTMLElement."